Various design and color updates

schu opened the pull request:
Various design and color updates, most importantly a different color palette and updates to the design of the question and comment sections. Details in the commit messages. This is a work in progress.

levinuss commented on resources/js/SessionQuestionView.svelte:
```diff
@@ -105,7 +105,9 @@
     {:else}
-        <div id="question{question.id}">
+        <div id="question{question.id}"
+            class="px-2 py-3 border rounded-3 shadow-sm bg-light-subtle"
```
`py-2` instead of `py-3` here?

levinuss reviewed (changes requested):
I like the overall look&feel of this!
See comments for some suggestions and bugs.
Are you aware that in the single question view (`question.blade.php`) much more width is used than in the session view now? This is inconsistent but could also be fine I think.

schu commented on resources/js/SessionAnswerView.svelte:
```diff
@@ -23,7 +23,7 @@
 
     $: if (answer) {
         markedAs = '';
```
Fixed, if I didn't miss a condition, please double-check.

schu commented on resources/js/SessionAnswerView.svelte:
```diff
@@ -118,8 +125,8 @@
 
 <div class="row ms-1 mb-2 me-1">
     {#if !examMode && answer.hint && (questionIsAnswered || answerContext.isSelectedAnswer)}
-        <div class="col-1 border-3 border-start border-secondary-subtle" />
```
I also think the design is not great, but I would like to leave this for later.

schu commented:
> Are you aware that in the single question view (`question.blade.php`) much more width is used than in the session view now?

We currently use the full container width in the single question view where we have a dynamic width in the session view, yes.

But I updated `deck.blade.php` and `deck-question.blade.php` to take less horizontal space (not the full sceen width).

I also added a commit `resources/views/deck: make deck editor and settings buttons float-end` as an addendum for 89dd9412d47f03ecd7e4f3411db6e9fe20973c8e

Please take another look.

levinuss reviewed (approved):
Everything resolved, looks good now.

Changes to the deck editor / settings buttons also make sense.

Thanks!
